File: GameJamProject/Assets/scripts/Falling.js
﻿#pragma strict

var platform : GameObject;

var sound : AudioClip;
private var hasPlayed = false;

function OnTriggerEnter2D (other : Collider2D) {

   if(hasPlayed == false){
    GetComponent.<AudioSource>().PlayOneShot(sound);
    hasPlayed = true;
   }

  platform.GetComponent.<Rigidbody2D>().isKinematic = false;

   }

File: GameJamProject/Assets/scripts/Falling.cs
using UnityEngine;
using System.Collections;

public class Falling : MonoBehaviour
{

    public float fallDelay = 1f;


    private Rigidbody2D rb2d;

    void Awake()
    {
        rb2d = GetComponent<Rigidbody2D>();
    }

   

    void Fall()
    {
        rb2d.isKinematic = false;
    }

    void Update(Collision2D other)
    {
        if (other.gameObject.CompareTag("Player"))
        {
            Invoke("Fall", fallDelay);
        }
    }
}